src/endpoints/payment.ts

```diff
@@ -8,12 +8,15 @@ import {
   CreatePaymentRequestBody,
   IdealPaymentData,
   IdealPaymentRequestBody,
+  GoogleOrApplePaymentData,
+  GoogleOrApplePaymentRequestBody,
   ValidateReceiptData,
   ReceiptValidationPlatform,
   CreateDonationPaymentData,
   CreateDonationPaymentRequestBody,
   ConfirmDonationPaymentData,
   ConfirmDonationPaymentRequestBody,
+  StripePaymentPaymentMethods,
```

Comment:
Is there any particular reason why Payment is doubled in the name?

Reply:
Well spotted, I didn't notice this. :) But the reasoning behind this "doubling" is this: It's an enum of possible payment methods that a Stripe payment can use. So Stripe payment's payment methods. When we say just "methods" we usually mean "an object's method functions", so that's why I'm clarifying it's a "payment method" we're talking about. I'll change it if you insist and I'm open for suggestions.

Reply:
I don't know if it is suitable, but maybe it can be named only `StripePaymentMethods`? 
Or maybe someone else from @inplayer-org/frontend has another idea.